src/websrc/json/en.json

```diff
@@ -328,7 +328,7 @@
     "cc": "Copy to clipboard",
     "erss": "Error saving settings. Check your network",
     "ercn": "Error. Check your network",
-    "proj": "Created with ❤️ by",
+    "proj": "Designed with ❤️ by",
```

Comment:
I prefer to use a verb "create" cause creation is not only design.

Reply:
The concept "**designed**" already contains a logical conclusion about creation, i.e. the use of the word _Design_ is a unique form indicating that development, creation, design and etc... were already included ova here. :+1: 

But again, I’m just suggesting with emphasis, I have no desire to impose, u have the right to demand what u like, I just want to add a fresher look. :wink:

Reply:
I won’t give arguments here, I just “want” to leave “created”. 😆